service/matching/task_queue_partition_manager.go

```diff
@@ -61,14 +61,16 @@ type (
 		matchingClient      matchingservice.MatchingServiceClient
 		metricsHandler      metrics.Handler // namespace/taskqueue tagged metric scope
 		// TODO(stephanos): move cache out of partition manager
-		cache cache.Cache // non-nil for root-partition
+		cache      cache.Cache // non-nil for root-partition
+		autoEnable bool
```

Comment:
maybe we shouldn't copy this into the struct but just read it when we need?

I'm thinking along the lines of: we find a bug with v2 that makes us want to disable auto-enable to prevent further problems. if we've copied it into the struct, that disabling won't take effect on loaded task queues unless we reload everything.